src/CMakeLists.txt

```
@@ -120,5 +121,31 @@ if(WIN32)
 elseif(APPLE)
     target_link_libraries(prince SDL2main SDL2 SDL2_image m)
 else() # Linux, *BSD, etc.
-    target_link_libraries(prince SDL2 SDL2_image m)
+    if(PSP)
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3")
+    	target_include_directories(prince PRIVATE ${SDL2_INCLUDE_DIRS} ${PSPDEV}/psp/include)
+    	target_link_libraries(prince PRIVATE
+    	${SDL2_LIBRARIES}
+    	z
+    	jpeg
+    	SDL2main
+    	SDL2
+    	SDL2_image
+    	vorbis
+    	vorbisfile
+    	ogg
+    	modplug
+    	png
+    	m
+    )
```

I would use pkgconf here, making is much more readable
```suggestion
include(FindPkgConfig)
pkg_search_module(SDL2 REQUIRED sdl2)

target_include_directories(target PRIVATE ${SDL2_INCLUDE_DIRS})

target_link_libraries(target PRIVATE
    ${SDL2_LIBRARIES}
)
```